src/js/components/Welcome/Footer.jsx

```diff
@@ -125,16 +125,9 @@ class Footer extends Component {
               color="default"
               variant="outlined"
               classes={{ root: classes.buttonOutlined }}
+              onClick={() => window.open('https://help.wevote.us/hc/en-us/requests/new', '_blank')}
             >
```

Comment:
Hi @jarodpeachey, in general we don't want to directly open external links like this because it makes the page not compatible with Cordova, the system we use to make our mobile apps. We can get away with this here though since we don't offer the Welcome pages within our iOS and Android apps.